packages/react-next/src/components/Callout/Callout.types.ts

```diff
@@ -1,14 +1,10 @@
 import * as React from 'react';
 import { IStyle, ITheme } from '../../Styling';
 import { DirectionalHint } from '../../common/DirectionalHint';
-import { Point, IRectangle, IStyleFunctionOrObject } from '../../Utilities';
+import { IRectangle, IStyleFunctionOrObject } from '../../Utilities';
 import { ICalloutPositionedInfo } from 'office-ui-fabric-react/lib/utilities/positioning';
 import { ILayerProps } from '../../Layer';
-
-/**
- * {@docCategory Callout}
- */
-export type Target = Element | string | MouseEvent | Point | null | React.RefObject<Element>;
+import { Target } from '@uifabric/react-hooks';
```

Comment:
Even for react-next I do wonder if we should continue exporting Target, especially since we don't re-export react-hooks.